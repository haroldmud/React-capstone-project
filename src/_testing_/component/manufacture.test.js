import React from 'react';
import renderer from 'react-test-renderer';
import Manufactures from '../../_mock_/components/manufacture';

it('component render correctely', () => {
  const tree = renderer
    .create(<Manufactures />)
    .toJSON();
  expect(tree).toMatchSnapshot();
});
