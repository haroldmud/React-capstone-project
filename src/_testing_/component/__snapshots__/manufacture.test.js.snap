// Jest Snapshot v1, https://goo.gl/fbAQLP

exports[`component render correctely 1`] = `
<div
  className="mainPage"
>
  <section
    className="cardCarrier"
  />
  <div
    className="pageButtons"
  >
    <h4>
      1
       
      of
       
      0
    </h4>
    <div
      className="btn"
    >
      <button
        className="previous"
        onClick={[Function]}
        type="button"
      >
        &lt;
      </button>
      <button
        className="next"
        onClick={[Function]}
        type="button"
      >
        &gt;
      </button>
    </div>
  </div>
</div>
`;
